packages/react-reconciler/src/ReactFiberBeginWork.old.js

```diff
@@ -2345,6 +2349,7 @@ function mountSuspenseFallbackChildren(
     );
   }
 
+  primaryChildFragment.stateNode = primaryChildInstance;
```

Comment:
I think you also need to do this in `updateOffscreenComponent`, because once `<Offscreen />` is a public API, it won't have been rendered by a Suspense parent that initializes the instance here.

But it might be best to do it in `createOffscreenFiber`:

https://github.com/facebook/react/blob/cb1e7b1c6ccd989d54b376ee4ae9da72a34f96e5/packages/react-reconciler/src/ReactFiber.new.js#L721-L731

That's where HostPortal initializes its instance